transit_odp/organisation/querysets.py

```diff
@@ -346,13 +345,9 @@ def add_pretty_status(self):
         return self.annotate(
             status=Case(
                 When(
-                    Q(live_revision__status="live"),
+                    Q(live_revision__status__in=["live", "error"]),
                     then=Value("published", output_field=CharField()),
                 ),
-                When(
-                    Q(live_revision__status="error", dataset_type=AVLType),
-                    then=Value(NO_ACTIVITY, output_field=CharField()),
-                ),
```

Comment:
Now looking at this again, the second when condition should be kept because it's only for AVL that we cant the status "error" to be shown as "published", and not for timetables/fares